Sessionless Product Detail.md

```diff
@@ -0,0 +1,50 @@
+# Sessionless Product Details
+
+**Introduction & Base Definitions**
+
+
+Authentication protocols (auth protocols) are gatekeepers that verify the identity of a person to give them access to specific programs or data. They include a set of rules or procedures that govern how data is transmitted across electronic devices. The most common uses of auth protocols are passwords, but others include one-time-passwords, biometrics, and magic links. 
+
+- An Application Programming Interface (API) is  a software intermediary that allows two  applications to share data with each other
+- An API Key is  a unique code that identifies an application or user
+- An API Token is a piece of code that contains information about a user and is used to secure access to an API
```

Comment:
This isn't quite correct. In most cases API Key and API Token are synonymous. A JWT (JSON Web Token) does contain information about a user as you've described here

Reply:
Updated with latest merge